v1.1/licensing.md

```diff
@@ -0,0 +1,94 @@
+- Enterprise license: An enterprise license enables you to use CockroachDB enterprise features for longer periods (one year or more).
+
+## Request a Trial or an Enterprise License Key
+
+[Contact us](https://www.cockroachlabs.com/pricing/sales/) to request a trial or an enterprise license key.
+
+## Set the Trial or Enterprise License Key
+
+As the CockroachDB `root` user, open the [built-in SQL shell](use-the-built-in-sql-client.html) in the insecure or secure mode, as per your CockroachDB setup. In the following example, we assume that CockroachDB is running in the insecure mode. Then use the `SET CLUSTER SETTING` command to set the name of your organization and the license key:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+>  SET CLUSTER SETTING cluster.organization = 'Acme Company'; 
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+>  SET CLUSTER SETTING enterprise.license = 'xxxxxxxxxxxx’;
+~~~
+
+## Verify the License Key is Set
+
+To verify the license key is set, open the [built-in SQL shell](use-the-built-in-sql-client.html) and use the `SHOW CLUSTER SETTING` command to verify the organization name and license key:
+
+{% include copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING cluster.organization;
+~~~
+~~~
++----------------------+
+| cluster.organization |
++----------------------+
+| Acme Company         |
++----------------------+
+(1 row)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING enterprise.license;
+~~~
+~~~
++--------------------------------------------------------------------+
+|                         enterprise.license                         |
++--------------------------------------------------------------------+
+| xxxxxxxxxxxx                                                       |
++--------------------------------------------------------------------+
+(1 row)
+~~~
+
+The license setting is also logged in the cockroach.log on the node where the command is run:
+
+{% include copy-clipboard.html %}
+~~~ sql
+$ cat cockroach.log | grep license
+~~~
+~~~
+I171116 18:11:48.279604 1514 sql/event_log.go:102  [client=[::1]:56357,user=root,n1] Event: "set_cluster_setting", target: 0, info: {SettingName:enterprise.license Value:xxxxxxxxxxxx User:root}
+~~~
+
+## Renew Expired License
+
+After your license expires, the enterprise features stop working, but your production setup is unaffected. For instance, after the license expires, the backup and restore features won't work until the license is renewed. But you can continue using all other features of CockroachDB without interruption.
```

Comment:
I'd tighten the second and third sentences a bit:

> For example, the backup and restore features would not work until the license is renewed, but you would be able to continue using all other features of CockroachDB without interruption.

Reply:
Fixed.